wrappers/wine-msvc.sh

```diff
@@ -80,4 +81,28 @@ else
 	sed -e "$dos2unix" -e "$unixify_path" -e "$unixify_line" <$WINE_MSVC_STDOUT &
 	sed -e "$dos2unix" -e "$unixify_path" <$WINE_MSVC_STDERR >&2 &
 	WINEDEBUG=-all wine64 "$EXE" "${ARGS[@]}" &>/dev/null
+	ec=$?
 fi
+
+[ $ec -ne 0 ] && exit $ec
+
+# Postprocess
+for a in "${ARGS[@]}"; do
+	case $a in
+		[-/]P) arg_P=$a ;;
+		[-/]Fi[zZ]:*) a=${a:5}; arg_Fi=${a//\\//} ;;
+		[-/]Fi*) a=${a:3}; arg_Fi=${a//\\//} ;;
+	esac
+done
+
+# Unixify paths for cl /P
+if [ -n "$arg_P" ] && [ -f "$arg_Fi" ]; then
+	if sed --help 2>&1 | grep '\-i extension' >/dev/null; then
```

Comment:
Oh, I hadn't expected that way of handling this. I guess this works, although it feels somewhat brittle.

Personally I just tend to avoid the `-i` form due to this ambiguity; I just pipe to a temp file and rename it afterwards. Not very elegant, but robust at least.

In any case, either form should be fine as long as we keep testing both platforms.

Reply:
I like `sed -i` for the reason that it supports multiple files.

`/P` may outputs multiple `.i` files if multiple source files are provided. In this case we can simply replace `arg_Fi` with an array. Though this PR doesn't handle this case.